src/ui/popupmenu/completion_item_widget.rs

```diff
@@ -37,11 +40,23 @@ impl CompletionItemWidgetWrap {
         let grid = gtk::Grid::new();
         grid.set_column_spacing(10);
 
-        let buf = get_icon_pixbuf(&item.kind.as_str(), icon_fg, size);
-        let kind = gtk::Image::new_from_pixbuf(&buf);
-        kind.set_tooltip_text(format!("kind: '{}'", item.kind).as_str());
-        kind.set_margin_start(margin);
-        grid.attach(&kind, 0, 0, 1, 1);
+        let buf = get_icon_pixbuf(&item.kind, icon_fg, size);
+        let image = match &item.kind {
```

Comment:
Wouldn't this do the trick?
```rust
let image = gtk::Image::new();
if show_kind {
    let buf = ....;
    image.set_from_pixbuf(&buf);
}
```

We either show the kind or we don't - no matter what the kind type is.

Reply:
Great point, didn't think about that. Thanks!